plugins/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ SET(CMAKE_DEBUG_POSTFIX "")
 # Enable C++17
 SET(CMAKE_CXX_STANDARD 17)
 
-IF(LMMS_BUILD_APPLE)
+IF(LMMS_BUILD_APPLE AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
Hmm.... I think it's `AppleClang`, e.g:

https://github.com/LMMS/lmms/blob/c53f3d981b9710bb1601e9213f925d95bdd9bd72/cmake/modules/CompileCache.cmake#L11

Per: https://cmake.org/cmake/help/latest/policy/CMP0025.html

Reply:
`MATCHES` covers that (`STREQUAL` would not). Earlier CMake policy will use Clang for Apple Clang, so it is not sure we have AppleClang as an ID. Unless the policy is enforced here, and then either is fine.